service/src/main/kotlin/no/nav/su/se/bakover/service/behandling/BehandlingServiceImpl.kt

```diff
@@ -1,361 +0,0 @@
-                tilordnetRessurs = behandlingTilAttestering.attestering()?.attestant
-            )
-        ).getOrElse {
-            log.error("Kunne ikke opprette Attesteringsoppgave. Avbryter handlingen.")
-            return KunneIkkeSendeTilAttestering.KunneIkkeOppretteOppgave.left()
-        }.also {
-            behandlingMetrics.incrementTilAttesteringCounter(BehandlingMetrics.TilAttesteringHandlinger.OPPRETTET_OPPGAVE)
-        }
-        behandlingTilAttestering.oppdaterOppgaveId(nyOppgaveId)
-        behandlingRepo.oppdaterOppgaveId(behandlingTilAttestering.id, nyOppgaveId)
-        behandlingRepo.settSaksbehandler(behandlingId, saksbehandler)
-        behandlingRepo.oppdaterBehandlingStatus(behandlingId, behandlingTilAttestering.status())
-        behandlingMetrics.incrementTilAttesteringCounter(BehandlingMetrics.TilAttesteringHandlinger.PERSISTERT)
-
-        oppgaveService.lukkOppgave(eksisterendeOppgaveId).map {
-            behandlingMetrics.incrementTilAttesteringCounter(BehandlingMetrics.TilAttesteringHandlinger.LUKKET_OPPGAVE)
-        }.mapLeft {
-            log.error("Klarte ikke å lukke oppgave. kall til oppgave for oppgaveId ${behandlingTilAttestering.oppgaveId()} feilet")
-        }
-        return behandlingTilAttestering.let {
-            observers.forEach { observer -> observer.handle(Event.Statistikk.BehandlingTilAttestering(it)) }
-            it.right()
-        }
-    }
-
-    override fun iverksett(
-        behandlingId: UUID,
-        attestant: NavIdentBruker.Attestant
-    ): Either<KunneIkkeIverksetteBehandling, IverksattBehandling> {
-        return iverksettBehandlingService.iverksett(behandlingId, attestant)
-    }
-
-    override fun ferdigstillInnvilgelse(behandling: Behandling) {
-        // TODO: delme: return ferdigstillIverksettingService.ferdigstillInnvilgelse(behandling)
-    }
-
-    override fun opprettSøknadsbehandling(
-        søknadId: UUID
-    ): Either<KunneIkkeOppretteSøknadsbehandling, Behandling> {
-        val søknad = søknadService.hentSøknad(søknadId).getOrElse {
-            return KunneIkkeOppretteSøknadsbehandling.FantIkkeSøknad.left()
-        }
-        if (søknad is Søknad.Lukket) {
-            return KunneIkkeOppretteSøknadsbehandling.SøknadErLukket.left()
-        }
-        if (søknad !is Søknad.Journalført.MedOppgave) {
-            // TODO Prøv å opprette oppgaven hvis den mangler? (systembruker blir kanskje mest riktig?)
-            return KunneIkkeOppretteSøknadsbehandling.SøknadManglerOppgave.left()
-        }
-        if (søknadRepo.harSøknadPåbegyntBehandling(søknad.id)) {
-            // Dersom man legger til avslutting av behandlinger, må denne spørringa spesifiseres.
-            return KunneIkkeOppretteSøknadsbehandling.SøknadHarAlleredeBehandling.left()
-        }
-        val nySøknadsbehandling = NySøknadsbehandling(
-            id = UUID.randomUUID(),
-            opprettet = Tidspunkt.now(clock),
-            sakId = søknad.sakId,
```

Comment:
Vurder modell for inn/ut for å unngå hack med saksnummer.

Reply:
Kan gjøres senere (ref annen kommentar)